nautobot/core/settings.py

```diff
@@ -725,7 +726,9 @@
 CELERY_BROKER_URL = os.getenv("NAUTOBOT_CELERY_BROKER_URL", parse_redis_connection(redis_database=0))
 
 # Celery results backend URL to tell workers where to publish task results
-CELERY_RESULT_BACKEND = os.getenv("NAUTOBOT_CELERY_RESULT_BACKEND", parse_redis_connection(redis_database=0))
+CELERY_RESULT_BACKEND = os.getenv(
+    "NAUTOBOT_CELERY_RESULT_BACKEND", "nautobot.core.celery.backends.NautobotDatabaseBackend"
+)
```

Comment:
Do we still even want to keep this setting configurable by environment variable?

Reply:
Yeah great question. This was provided before due to varying Redis instances. We should just remove it.

Reply:
But we'll also need to update the docs and put it into the release notes/migration guides that this value shall not be overridden anymore due to Jobs overhaul, etc, etc.